<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Home</class>
 <widget class="QDialog" name="Home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>1080</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <widget class="QWidget" name="profile">
    <property name="maximumSize">
     <size>
      <width>1916</width>
      <height>16777215</height>
     </size>
    </property>
    <attribute name="title">
     <string>Profile</string>
    </attribute>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>1790</x>
       <y>10</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Log Out</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>70</y>
       <width>1281</width>
       <height>851</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="background.qrc">:/background/playground.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1920</width>
       <height>1080</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="background.qrc">:/background/bg.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>231</width>
       <height>181</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <zorder>label_2</zorder>
    <zorder>pushButton</zorder>
    <zorder>label</zorder>
    <zorder>label_4</zorder>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Players</string>
    </attribute>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1920</width>
       <height>1080</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="background.qrc">:/background/bg.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Leader Board</string>
    </attribute>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1920</width>
       <height>1080</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="background.qrc">:/background/bg.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <property name="minimumSize">
     <size>
      <width>1916</width>
      <height>1049</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1916</width>
      <height>1049</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <attribute name="title">
     <string>Rules</string>
    </attribute>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>310</y>
       <width>751</width>
       <height>421</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="background.qrc">:/background/IMG-20230415-WA0008.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1920</width>
       <height>1080</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="background.qrc">:/background/bg.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <zorder>label_7</zorder>
    <zorder>label_5</zorder>
   </widget>
   <widget class="QWidget" name="Admin">
    <attribute name="title">
     <string>Admin</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1921</width>
       <height>1051</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox
{
	border: none;
}
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="ctl_users">
      <attribute name="title">
       <string>CONTROL USERS</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>130</y>
         <width>431</width>
         <height>91</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pb_read">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>READ</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_delete">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>DELETE</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTableWidget" name="tableWidget">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>230</y>
         <width>1011</width>
         <height>521</height>
        </rect>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>130</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string>ID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>User Name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Points</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Budget</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Is Admin</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Club ID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Email</string>
        </property>
       </column>
      </widget>
      <widget class="QPushButton" name="pb_insert">
       <property name="geometry">
        <rect>
         <x>1620</x>
         <y>130</y>
         <width>156</width>
         <height>89</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>INSERT</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_update">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>120</y>
         <width>191</width>
         <height>81</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>UPDATE</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="gb_user_update">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>230</y>
         <width>431</width>
         <height>511</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLineEdit" name="le_target_id">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>341</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="placeholderText">
         <string>target id</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="sb_budget_update">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>190</y>
          <width>211</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="cb_club_update">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>310</y>
          <width>341</width>
          <height>24</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Liverpool</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Chelsea</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ManchesterCity</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Pyramids</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Alahly</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Zamalek</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Barcelona</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>RealMadrid</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>AtleticoMadrid</string>
         </property>
        </item>
       </widget>
       <widget class="QPushButton" name="pb_conform_update">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>420</y>
          <width>131</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>Conform</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_password_update">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>150</y>
          <width>341</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="placeholderText">
         <string>password</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_budget_update">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>190</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>budget</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_points_update">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>230</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>points</string>
        </property>
       </widget>
       <widget class="QLabel" name="l_choose_club_update">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>270</y>
          <width>341</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Choose Club</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_username_update">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>70</y>
          <width>341</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="placeholderText">
         <string>username</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="sb_points_update">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>230</y>
          <width>211</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_email_update">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>110</y>
          <width>341</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="placeholderText">
         <string>email</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="is_admin_update">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>360</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Admin ?</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_user_update">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>30</y>
          <width>80</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>update</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="gb_user_insert">
       <property name="geometry">
        <rect>
         <x>1470</x>
         <y>220</y>
         <width>431</width>
         <height>511</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLineEdit" name="le_username_insert">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>50</y>
          <width>341</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="placeholderText">
         <string>username</string>
        </property>
       </widget>
       <widget class="QLabel" name="l_choose_club">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>250</y>
          <width>341</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Choose Club</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="sb_budget_insert">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>170</y>
          <width>211</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_points_insert">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>210</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>points</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_budget_insert">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>170</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>budget</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_conform_insert">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>380</y>
          <width>131</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>Conform</string>
        </property>
       </widget>
       <widget class="QComboBox" name="cb_club_insert">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>290</y>
          <width>341</width>
          <height>24</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Liverpool</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Chelsea</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ManchesterCity</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Pyramids</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Alahly</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Zamalek</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Barcelona</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>RealMadrid</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>AtleticoMadrid</string>
         </property>
        </item>
       </widget>
       <widget class="QSpinBox" name="sb_points_insert">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>210</y>
          <width>211</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QCheckBox" name="is_admin_insert">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>340</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Admin ?</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_password_insert">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>130</y>
          <width>341</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="placeholderText">
         <string>password</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_email_insert">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>90</y>
          <width>341</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="placeholderText">
         <string>email</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="gb_user_delete">
       <property name="geometry">
        <rect>
         <x>870</x>
         <y>80</y>
         <width>331</width>
         <height>141</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="l_enter_id">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Eenter ID</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_enter_id">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>161</width>
          <height>71</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_conform">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>60</y>
          <width>131</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>Conform</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="ctl_players">
      <attribute name="title">
       <string>CONTROL PLAYERS</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="ctl_matches">
      <attribute name="title">
       <string>CONTROL MATCHES</string>
      </attribute>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="background.qrc"/>
 </resources>
 <connections/>
</ui>
